### PR TITLE
Task #171054 Chore: Resolving php 7.4 Notices

### PR DESCRIPTION
**Pull request by divyachaudhari**
Following Notices were displayed:

Notice: Undefined index: JHtmlBootstrap::startTabSet in C:\xampp\htdocs\JT\libraries\cms\html\bootstrap.php on line 798
Notice: Undefined property: Joomla\CMS\Object\CMSObject::$mediaData in C:\xampp\htdocs\JT\administrator\components\com_tjcertificate\views\trainingrecord\tmpl\edit.php on line 80

### CODE REVIEW

File: src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php [hunk 1]
```diff
@@ -52,7 +52,6 @@
 		?>
 		<div class="form-horizontal">
 
-		<?php echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
```
Comment (praneettekdi) [inline src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php:55]: Why is this removed? @divyachaudhari

File: src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php [hunk 2]
```diff
@@ -51,15 +51,15 @@
 		}
 		?>
 		<div class="form-horizontal">
-
-		<?php echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
+		<?php
+			echo HTMLHelper::_('bootstrap.startTabSet', 'myTab', array('active' => 'general'));
+			echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
```
Comment (praneettekdi) [inline src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php:56]: Now you have added two new sets of code. @divyachaudhari 
Is there anything you fixed here? else keep it is as it was earlier

Comment (divyachaudhari) [reply src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php:56]: For the tab, The warning were dsiplayed and the "Issues Certificate" tab was not displayed due to the warnings. This code has removed the warning and also displaying the tab "Issued certificate"
